node/hack/query/send_req.go

```diff
@@ -483,6 +510,8 @@ func sendSolanaQueryAndGetRsp(queryRequest *query.QueryRequest, sk *ecdsa.Privat
 					switch r := response.PerChainResponses[index].Response.(type) {
 					case *query.SolanaAccountQueryResponse:
 						logger.Info("solana query per chain response", zap.Int("index", index), zap.Any("pcr", r))
```

Comment:
Did you want to make the text in the Info "solana account query..." and the one on line 514 "solana pda query..." to distinguish them?